### PR TITLE
SeaORM 0.11.x Docs - 1

### PR DESCRIPTION
**Pull request by billy1624**
We should execute the following command to tag 0.11.x docs before merging this into master

```sh
npm run docusaurus docs:version 0.11.x
```

REF: https://docusaurus.io/docs/versioning#directory-structure

### CODE REVIEW

**Review by billy1624 — COMMENTED** [review]
Done! I'll write the SeaORM 0.11.0 release blog on another PR

@tyt2y3 please review and merge this :)

File: SeaORM/docs/02-install-and-config/02-connection.md [hunk 1]
```diff
@@ -34,3 +34,14 @@ opt.max_connections(100)
 
 let db = Database::connect(opt).await?;
 ```
+
+## Closing Connection
+
+To close the connection explicitly, call the `close` method.
```
Comment (tyt2y3) [inline SeaORM/docs/02-install-and-config/02-connection.md:40]:
```suggestion
The connection will be automatically closed on drop. To close the connection explicitly, call the `close` method.
```

File: SeaORM/docs/04-generate-entity/02-entity-structure.md [hunk 2]
```diff
@@ -162,6 +162,18 @@ If you want to ignore a particular model attribute such that it maps to no datab
 pub ignore_me: String
 ```
 
+### Cast Column Type on Select and Save
+
+If you need to select a column as one type but save it into the database as another, you can specify the `select_as` and the `save_as` attributes to perform the casting. A typical use case is selecting a column of type `citext` (case-insensitive text) as `String` in Rust and saving it into the database as `citext`. One should define the model field as below:
+
+```rust
+#[sea_orm(
```
Comment (tyt2y3) [inline SeaORM/docs/04-generate-entity/02-entity-structure.md:170]: Can put these on one line?

File: SeaORM/docs/05-basic-crud/03-insert.md [hunk 3]
```diff
@@ -16,6 +16,20 @@ let _: ActiveValue<i32> = Set(10);
 let _: ActiveValue<i32> = NotSet;
 ```
 
+### Reset ActiveValue
+
+Calling `reset` method will convert `Unchanged` active value as `Set`.
```
Comment (tyt2y3) [inline SeaORM/docs/05-basic-crud/03-insert.md:21]:
```suggestion
Calling `reset` method will convert `Unchanged` active value as `Set`, thus the attribute will be saved in the next operation.
```

File: SeaORM/docs/04-generate-entity/02-entity-structure.md (hunk 2 above)
Comment (billy1624) [reply SeaORM/docs/04-generate-entity/02-entity-structure.md]: Sure, that's more compact :)

**Comment by billy1624** [comment]
Updated

**Pull request merged**